nvm.sh

```diff
@@ -417,7 +416,7 @@ nvm_ensure_version_installed() {
     else
       local PREFIXED_VERSION
       PREFIXED_VERSION="$(nvm_ensure_version_prefix "${PROVIDED_VERSION}")"
-      nvm_err "N/A: version \"${PREFIXED_VERSION:-$PROVIDED_VERSION}\" is not yet installed."
+      nvm_err "N/A: version \"${PREFIXED_VERSION:-${PROVIDED_VERSION}}\" is not yet installed."
```

Comment:
i think i tried this in the past and it failed, which is why this was done this way.

Please also test in `ksh`, which we don't test in CI but it still needs to work in.

Reply:
Okay, so I'll rollback this line, thanks!